shell/update-counts.sh

```diff
@@ -52,7 +45,4 @@ update_counts() {
 update_counts "Segments" "segmentCountMap"
 update_counts "Tranches" "trancheCountMap"
 
-# Reformat the code with Forge
-forge fmt $BASE_SCRIPT
```

Comment:
In that case, we can keep it.

Reply:
Addressed: https://github.com/sablier-labs/v2-core/pull/931/commits/2a3b509fe359ca20ee3a0a88cbea066b0bd60d67